doc/attributes_mathMap.dox

```diff
@@ -2,7 +2,7 @@ namespace ast {
 /**
 @page mathmap_attributes MathMap Attributes
 
-@section MathMap_Attributes @ref MathMap Attributes
+@section MathMap_Attributes MathMap Attributes
```

Comment:
I agree that the extra `@section` text looks funny and I would prefer to get rid of it if I could do it without a lot of work. However, the reason it is there is so I can use `@subsection` for each attribute name, to start the subsection describing that attribute. This is a simple way to give me an anchor I can link to, and a nicely formatted header for the description of each attribute.

Unfortunately, `@subsection` must be in a `@section`.

The only simple alternative I have thought of is to delete the existing `@section` commands and change all the `@subsection` commands to `@section` commands. But that makes the text for each attribute name larger than I'd like.

I think it looks a bit strange either way, so I'm leaving it for now. If you have any suggestions, please let me know.